sdk/identity/identity/TOKEN_CACHING.md

```diff
@@ -21,6 +21,10 @@ The in-memory token cache provided by the Azure Identity library:
 
 **Note:** When Azure Identity library credentials are used with Azure service libraries (for example, Azure Blob Storage), the in-memory token caching is active in the `HttpPipeline` layer as well. All `TokenCredential` implementations are supported there, including custom implementations external to the Azure Identity library.
 
+#### Caching cannot be disabled
+
+As there are many levels of cache it is not possible to clear the cache.
```

Comment:
I assume that creating a new credential instance will clear the cache. See https://github.com/Azure/azure-sdk-for-java/pull/36304#discussion_r1293535580. If so, add that detail here too.

Reply:
done